install/streams.csv

```diff
@@ -1686,7 +1686,8 @@ PTAZ,10,H,H,100,false,false,false,2022-12-08T21:30:00Z,9999-01-01T00:00:00Z
 PTHS,20,B,N,50,true,false,false,2014-08-18T23:01:00Z,2020-06-16T00:00:00Z
 PTHS,20,H,N,200,true,false,true,2002-06-21T00:00:00Z,9999-01-01T00:00:00Z
 PTOS,20,B,N,50,true,false,false,2011-06-03T04:00:00Z,2020-06-16T00:00:00Z
-PTOS,20,H,N,200,true,false,true,2011-06-03T04:00:00Z,9999-01-01T00:00:00Z
+PTOS,20,H,N,200,true,false,true,2011-06-03T04:00:00Z,2023-02-01T00:20:00Z
```

Comment:
llooks like it  is continuous since 2018.088 29th march 2018  
if ever @mnaguit

Reply:
Ei @salichon , thanks.
Should we align the entry based on continuous-triggered time, or based on this recorder swap?

Reply:
that s "optional" (informative in the response) 
Better to stick closer to the reality of the continuous streams in 2018 ?